3.txt

```diff
@@ -2,6 +2,8 @@ Third file
 New changes
 This line is changed in VM
 
+My name is Abhilash;.
+Hi I am causing merge conflict.
```

Comment:
Should I review this?

Reply:
Yes please review this. Unfortunately your code moved down